employees/pom.xml

```diff
@@ -31,7 +31,7 @@
     <dependencies>
         <dependency>
             <groupId>org.wildfly.swarm</groupId>
-            <artifactId>jaxrs-cdi</artifactId>
+            <artifactId>cdi</artifactId>
```

Comment:
Do we need the complete `cdi` module?

Reply:
javax.enterprise.context.ApplicationScoped in not available in jaxrs-cdi fraction anymore and hence I added the cdi-fraction instead